src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/VideoLinkMigrationServiceTest.kt

```diff
@@ -0,0 +1,335 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.whereabouts.model.HearingType
+import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkAppointment
+import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkBookingEvent
+import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkBookingEventType
+import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkAppointmentRepository
+import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingEventRepository
+import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingRepository
+import uk.gov.justice.digital.hmpps.whereabouts.services.court.VideoLinkMigrationService
+import uk.gov.justice.digital.hmpps.whereabouts.utils.DataHelpers.Companion.makeVideoLinkBooking
+import java.lang.IllegalArgumentException
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.Optional
+
+private val videoLinkAppointmentRepository: VideoLinkAppointmentRepository = mock()
```

Comment:
A bit unusual to have these mock repo's sitting outside of the actual test class?

Reply:
Moved them inside now.